strawberryfields/apps/data.py

```diff
@@ -12,367 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 r"""

```

Comment:
I was confused and kept `data.py` because in `apps/__init__.py` we write:

`import strawberryfields.apps.data`

but `data` is a folder now. Does this command still work?

Reply:
Yes, it's fine for data to be a folder as it has an `__init__.py` which signifies that it is a module. See [here](https://docs.python.org/3/reference/import.html#regular-packages) for example.